core/chains/evm/gas/chain_specific.go

```diff
@@ -8,18 +8,18 @@ import (
 
 // chainSpecificIsUsable allows for additional logic specific to a particular
 // Config that determines whether a transaction should be used for gas estimation
-func chainSpecificIsUsable(tx evmtypes.Transaction, baseFee *assets.Wei, cfg Config) bool {
-	if cfg.ChainType() == config.ChainXDai {
+func chainSpecificIsUsable(tx evmtypes.Transaction, baseFee *assets.Wei, chainType config.ChainType, minGasPriceWei *assets.Wei) bool {
```

Comment:
Note: I chose to refactor out Config since I think that decouples things a bit more, but can also see an argument for that not being desirable here (due to the generic nature of the function).

In the spirit of KISS, I went ahead but please lmk if you'd like me to change it back.